docs/ops/deploy.md

```diff
@@ -25,9 +25,27 @@ Edit conf/titan.toml and set the pd-addrs
 pd-addrs="tikv://your-pd-addrs:port"
 ```
 
+### Enable Multi-tenancy
+First, set auth value in the 'server' section of conf/titan.toml
+
+```
+auth = "somepwd"
+```
+
+Then，use ./tools/token/token to gen a client token.
+
+```
+cd $GOPATH/src/github.com/meitu/titan/tools/token
+go build main.go -o titan-gen-client-token
+./titan-gen-client-token -key somepwd -namespace bbs
+```
+
+You get the token for client auth, for example: bbs-1543999615-1-7a50221d92e69d63e1b443
```

Comment:
Use 'Then' to begin this sentence and future tense is better. Example: Then you'll get the token...

Reply:
fixed